tools/sgen/input/FluentUIStyle.yml

```diff
@@ -420,7 +420,7 @@ MSFGhostButtonTokens extends MSFButtonTokens:
   borderColor: [ rest: $Colors.Neutral.clear, hover: $Colors.Neutral.clear, pressed: $Colors.Neutral.clear, selected: $Colors.Brand.shade10, disabled: $Colors.Neutral.clear ]
 
 AP_MSFListTokens:
-  backgroundColor: [ rest: $Colors.Background.neutral1, pressed: $Colors.Background.neutral5 ]
+  backgroundColor: $Colors.Background.neutral1
```

Comment:
is this intended?

Reply:
Yes. The pressed color was not being used, so removing it to save that space too.